Task20/ClassLibrary3/Tests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using OpenQA.Selenium.Chrome;
+
+namespace Task70_POM
+{
+    [TestFixture]
+    public class Tests
+    {
+        public IWebDriver Driver { get; set; }
+        public WebDriverWait Wait { get; set; }
+
+        [SetUp]
+        public void SetupTest()
+        {
+            
+            Driver = new ChromeDriver(@"D:\Automation");
+            Wait = new WebDriverWait(Driver, TimeSpan.FromSeconds(30));
+        }
+
+        [Test]
+        public void LoginUserTutBy()
+        {
+            PageObject tutby = new PageObject(Driver);
+            tutby.Navigate();
+            string username = "seleniumtests@tut.by";
+            string password = "123456789zxcvbn";
```

Comment:
Move these variables on the test beginning.

Reply:
Changes are made in the new commit